docsSource/sections/02-utilities.md

```diff
@@ -7,17 +7,17 @@ Animate.css comes packed with a few utility classes to simplify its use.
 You can add delays directly on the element's class attribute, just like this:
 
 ```html
-<div class="animate__animated animate__bounce animate__delay-2s">Example</div>
+<div class="animate__animated animate__bounce animate__delay-2x">Example</div>
 ```
 
 Animate.css provides the following delays:
 
 | Class name          | Default delay time |
 | ------------------- | ------------------ |
-| `animate__delay-2s` | `2s`               |
-| `animate__delay-3s` | `3s`               |
-| `animate__delay-4s` | `4s`               |
-| `animate__delay-5s` | `5s`               |
+| `animate__delay-2x` | `2s`               |
+| `animate__delay-3x` | `3s`               |
+| `animate__delay-4x` | `4s`               |
+| `animate__delay-5x` | `5s`               |
```

Comment:
How about adding a notice right under this table about both sorts of classes?
```suggestion
| `animate__delay-5x` | `5s` |

> In the previous versions, the library supported a slightly different version of the classes, like `animate__delay-2s`. As this utility works as a multiplier, it made it very confusing to use if the root variable `--animate-delay` was set to anything but `1`. We changed it but we still support the old classes and there's no need to migrate your code to the newer version.

Reply:
Gotcha, @eltonmesquita!